src/renderer/api/terminal-api.ts

```diff
@@ -106,6 +109,17 @@ export class TerminalApi extends WebSocketApi {
     this.onReady.removeAllListeners();
   }
 
+  _onRcReady = (data: string) => {
+    if (!data) {
+      return;
+    }
+
+    if (data.match(/\r?\n/) === null && data.endsWith(" ")) {
```

Comment:
This is the most finicky part, basically I assume that all prompts will have a space and be only a single line.

Reply:
Need an explanation what are we doing here. Probably by moving to separate function with explanatory name or just by writing a comment.

Reply:
Sounds good, will do both.